pkg/actions/stop-all.go

```diff
@@ -52,3 +40,24 @@ func StopAllCommand() {
 		}
 	}
 }
+
+func getContainersToRemove(containerList []types.Container) []types.Container {
+	codewindContainerNames := []string{
+		"codewind-pfe",
+		"codewind-performance",
+	}
+
+	// Docker returns all the names with a "/" on the front
+	projectContainerPrefix := "/cw-"
```

Comment:
Doesn't that mean it should be `/codewind-pfe` and `/codewind-performance` above then?

Reply:
It doesn't in the current design but looking at the container names we use (codewind-pfe-amd64, codewind-performance-amd64) they can also use prefix rather than contains too.
I'll make that change

Reply:
@makandre See latest change.